tests/ci/run_fips_callback_tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -ex
```

Comment:
```suggestion
#!/usr/bin/env bash
set -ex
```

Don't hard-code the shell paths.